doc/ref/chap-builtin-func.md

```diff
@@ -20,6 +20,19 @@ This chapter in the [Oils Reference](index.html) describes builtin functions.
 
 ### type()
 
+Given an arbitrary value, returns a string representing the value's runtime type.
+
+For example:
+
+    var d = {'foo': 'bar'}
+    var n = 1337
+
+    $ echo $[type(d)]
```

Comment:
can you change these to the `=` style too ?   Also I guess it should be `$ var` on the first 2 lines to be consistent

Reply:
Ah whoops. Thought I got all of these